File: app/src/main/java/com/stevenfrew/beatprompter/midi/StartMessage.java
package com.stevenfrew.beatprompter.midi;

class StartMessage extends OutgoingMessage
{
    private final static byte[] MIDI_START_SIGNAL_BYTES=new byte[]{0x0f, Message.MIDI_START_BYTE,0,0};
    StartMessage()
    {
        super(MIDI_START_SIGNAL_BYTES,true);
    }
}

File: app/src/main/kotlin/com/stevenfrew/beatprompter/midi/StartMessage.kt
package com.stevenfrew.beatprompter.midi

internal class StartMessage : OutgoingMessage(MIDI_START_SIGNAL_BYTES, true) {
    companion object {
        private val MIDI_START_SIGNAL_BYTES = byteArrayOf(0x0f, Message.MIDI_START_BYTE, 0, 0)
    }
}